pkg/controller/installation/products/launcher/reconciler_test.go

```diff
@@ -0,0 +1,108 @@
+	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/marketplace"
+	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/products/config"
+	oauthv1 "github.com/openshift/api/oauth/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	fakeappsv1Client "github.com/openshift/client-go/apps/clientset/versioned/fake"
+	appsv1Client "github.com/openshift/client-go/apps/clientset/versioned/typed/apps/v1"
+	coreosv1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1"
+	operatorsv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	marketplacev1 "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"testing"
+)
+
+var (
+	integreatlyOperatorNamespace = "integreatly-operator-namespace"
+)
+
+func getBuildScheme() (*runtime.Scheme, error) {
+	scheme := runtime.NewScheme()
+	err := aerogearv1.SchemeBuilder.AddToScheme(scheme)
+	err = integreatlyv1alpha1.SchemeBuilder.AddToScheme(scheme)
+	err = operatorsv1alpha1.AddToScheme(scheme)
+	err = marketplacev1.SchemeBuilder.AddToScheme(scheme)
+	err = corev1.SchemeBuilder.AddToScheme(scheme)
+	err = coreosv1.SchemeBuilder.AddToScheme(scheme)
+	err = launcherv1alpha2.SchemeBuilder.AddToScheme(scheme)
+	err = routev1.SchemeBuilder.AddToScheme(scheme)
+	err = oauthv1.SchemeBuilder.AddToScheme(scheme)
+	return scheme, err
+}
+
+type LauncherTestScenario struct {
+	Name             string
+	Installation     *integreatlyv1alpha1.Installation
+	FakeSigsClient   client.Client
+	FakeAppsV1Client appsv1Client.AppsV1Interface
+	Manager          marketplace.MarketplaceInterface
+	ExpectedStatus   integreatlyv1alpha1.StatusPhase
+	Assert           AssertFunc
+}
+
+func TestLauncher(t *testing.T) {
+	scheme, err := getBuildScheme()
+	if err != nil {
+		t.Fatalf("Error creating build scheme")
+	}
+
+	scenarios := []LauncherTestScenario{
+		{
+			Name:             "Test successful installation without errors",
+			FakeSigsClient:   getSigClient(getClusterPreReqObjects(integreatlyOperatorNamespace), scheme),
+			FakeAppsV1Client: fakeappsv1Client.NewSimpleClientset(mockLauncherDeploymentConfigs...).AppsV1(),
+			Manager:          marketplace.NewManager(),
+			Installation: &v1alpha1.Installation{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-installation",
+					Namespace: integreatlyOperatorNamespace,
+				},
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: v1alpha1.SchemeGroupVersion.String(),
+				},
+				Spec: v1alpha1.InstallationSpec{
+					MasterURL:        "https://console.apps.example.com",
+					RoutingSubdomain: "apps.example.com",
+				},
```

Comment:
I think you just need to add a `Status.Phase: "Active"` to this namespace to fix your unit test.